extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -71,3 +77,32 @@ async function VerifyOeNode(server: TestServerProfile, timeout: number, expected
 	assert(expectedNodeLable.toLocaleString() === actualNodeLable.toLocaleString(), `Expected node label: "${expectedNodeLable}", Actual: "${actualNodeLable}"`);
 }
 
+async function VerifyDBContextMenu(server: TestServerProfile, timeout: number, expectedActions: string[]) {
+
+	await connectToServer(server, 3000);
+
+	let nodes = <azdata.objectexplorer.ObjectExplorerNode[]>await azdata.objectexplorer.getActiveConnectionNodes();
+	assert(nodes.length > 0, `Expecting at least one active connection, actual: ${nodes.length}`);
+
+	let index = nodes.findIndex(node => node.nodePath.includes(server.serverName));
+	assert(index !== -1, `Failed to find server: "${server.serverName}" in OE tree`);
+
+	let ownerUri = await azdata.connection.getUriForConnection(nodes[index].connectionId);
+	let dbName: string = 'TestDB_' + new Date().getTime().toString();
+	await createDB(dbName, ownerUri);
+
+	let serverNode = nodes[index];
+	let children = await serverNode.getChildren();
+	let databasesFolder = children[0]; // first should be databases
```

Comment:
it seems the test could be more robust here by checking whether the first node is databases folder.

Reply:
previous test is doing that - but added anyways to make sure